Tests/Resources/app/config/config.php

```diff
@@ -5,3 +5,10 @@
 
 $loader->import(CMF_TEST_CONFIG_DIR.'/default.php');
 $loader->import(CMF_TEST_CONFIG_DIR.'/phpcr_odm.php');
+
+$config = array(
+    'form' => array(
+        'csrf_protection' => false,
+    ),
+);
+$container->loadFromExtension('framework', $config);
```

Comment:
I do this since I am submitting a form in the `DocumentTypeTest.php` on line `73`. I don't think we need to have this enabled, so is it okay to disable this globally?